nixos/modules/services/networking/shorewall.nix

```
@@ -0,0 +1,71 @@
+{ config, lib, pkgs, ... }:
+let
+  types = lib.types;
+  cfg = config.services.shorewall;
+in {
+  options = {
+    services.shorewall = {
+      enable = lib.mkOption {
+        type        = types.bool;
+        default     = false;
+        description = ''
+          Whether to enable Shorewall Firewall.
+          *Warning*: Enabling this service WILL disable the existing NixOS
```

```suggestion
          WARNING: Enabling this service WILL disable the existing NixOS
```

The `*Text*` does not render so great [on website](https://nixos.org/nixos/options.html#services.ferm.enable) or in CLI

---

Since it's going to be rendered by DocBook for the official docs, you could also do this:
```suggestion
          <emphasis>Warning</emphasis>: Enabling this service WILL disable the existing NixOS
```

---

Actually, there is a `warning` element in docbook, which renders with a nice yellow box :)

```xml
<warning>
 <para>
  Enabling this service ...
 </para>
</warning>